client/src/context/GlobalState.js

```diff
@@ -174,6 +174,31 @@ export const GlobalProvider = ({ children }) => {
     }
   }
 
+  async function addFriend(userId, friend) {
+    const requestOptions = {
+      method: 'GET',
+      headers: {
+        ...authHeader(),
+        'Content-Type': 'application/json'
+      }
+    };
+
+    try {
+      const res = await axios.post(`/api/v1/users${userId}/friends`, friend, requestOptions);
```

Comment:
I saw you make a request to `/api/v1/users/${userId}/friends`, but I didn't see a corresponding backend part in this PR. Did you implement a backend function for this such a request?